src/components/Banners/SiteBanner.tsx

```diff
@@ -29,9 +30,17 @@ export const SiteBanner = () => {
     Cookies.set(text, "viewed", { expires: 7 });
   };
 
+  const jiraLinkify = (unlinkified: string) => {
+    const JIRA_REGEXP = /[A-Z]{1,10}-\d{1,6}/gi;
+    const linkified = unlinkified.replace(JIRA_REGEXP, function(match) {
+      return `<a href="https://${jiraHost}/browse/${match}">${match}</a>`;
+    });
+    return { __html: linkified };
+  };
+
   return showBanner ? (
     <Banner bannerTheme={theme} data-cy="sitewide-banner">
-      {text}{" "}
+      <span dangerouslySetInnerHTML={jiraLinkify(text)} />
```

Comment:
You can use the already installed `html-react-parser`  library here instead of `dangerouslySetInnerHtml`